pkg/storage/cassandra_dependencies.go

```diff
@@ -91,11 +91,17 @@ func cassandraDeps(jaeger *v1.Jaeger) []batchv1.Job {
 		})
 	}
 
-	annotations := map[string]string{
-		"prometheus.io/scrape":    "false",
-		"sidecar.istio.io/inject": "false",
-		"linkerd.io/inject":       "disabled",
+	truncatedName := util.Truncate("%s-cassandra-schema-job", 63, jaeger.Name)
```

Comment:
Iam not 100% sure, but somehow i have in mind there is a naming packe in place that we could use.